src/Magnum/OvrIntegration/Context.h

```diff
@@ -53,6 +53,17 @@ struct Error {
 };
 
 /**
+ * @brief Result of @ref Context::detect(int).
+ */
+enum class OvrDetectResultFlag: Byte {
+    ServiceRunning = 1,
+    HmdConnected = 2
+};
+
+typedef Corrade::Containers::EnumSet<OvrDetectResultFlag> OvrDetectResult;
+CORRADE_ENUMSET_OPERATORS(OvrDetectResult)
```

Comment:
@mosra Would you be okay with keeping the naming as is? Would you suggest just returning the same structure/creating an own equivalent structure instead of using enums?